### ISSUE TITLE
SentrySdk Configure scope does not work with serilog Log.Error

### ISSUE THREAD

**Issue by stany9g** [getsentry/sentry-dotnet#1530]
### Environment

How do you use Sentry?
Sentry.Serilog 3.15.0

Which SDK and version?
.NET 6

### Steps to Reproduce

1. Configure serilog with sentry
```
Log.Logger = new LoggerConfiguration()
                    .ReadFrom.Configuration(builder.Build())
                    .WriteTo.Sentry(o =>
                    {
                        o.MinimumBreadcrumbLevel = LogEventLevel.Debug; // Debug and higher are stored as breadcrumbs (default os Information)
                        o.MinimumEventLevel = LogEventLevel.Error; // Error and higher is sent as event (default is Error)
                                                               // If DSN is not set, the SDK will look for an environment variable called SENTRY_DSN. If nothing is found, SDK is disabled.
                        o.Dsn = "https://807eff2bb0bd4bb8bcfba5207e5af0c4@o818374.ingest.sentry.io/5868204";
                        o.AttachStacktrace = true;
                        o.SendDefaultPii = true; // send PII like the username of the user logged in to the device
                                                 // Optional Serilog text formatter used to format LogEvent to string. If TextFormatter is set, FormatProvider is ignored.
                        o.TextFormatter = new MessageTemplateTextFormatter("[{MyTaskId}] {Message}", null);
                        o.InitializeSdk = true;
                        // Other configuration
                    })
                    .CreateLogger();
```

2. Add scope in different place

```
var userName = "test";
                    var databaseName = "test";
                    var licenseId = "test";
                    SentrySdk.ConfigureScope(scope =>
                    {
                        scope.User = new User()
                        {
                            Username =  userName
                        };
                        scope.Contexts["Information"] = new
                        {
                            Database = databaseName,
                            License = licenseId
                        };
                    });
```

Thank you for the help.

### Result

When using Log.Error(ex, message) the exception with breadcrumb is succesfully send to sentry but without context information

**Comment by bruno-garcia** [comment]
What type of app is that? If the other place you refer to is another asynchronous operation, this is by design as the scopes are different. If you'd like to have a single, global scope (where all context data set in any thread goes in events sent by any thread) you can opt in to global mode but that usually only makes sense on desktop and cli tools

**Comment by stany9g** [comment]
Hello Bruno, the application is WPF desktop app so yes I would prefer one global context since the error could be logged in any Thread, but the important fact is that we know some custom info about that. I have found based on you tip about global mode in some other Github thread the **IsGlobalModeEnabled** option. I would suggest to add it to official documentation. Otherwise the issue was solved. Thank you for your help.
